src/app/comments/comment-form-container/comment-form-container.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="comment-form">
+  <h2>Add comment<span> - share your thought</span></h2>
+  <app-comment-form
+    (commentDataToParent)="getCommentDataFromParent($event)"
+    errorMessage="{{errorMessage}}"
+    successMessage="{{successfulSending}}">
```

Comment:
lets keep naming consistent with the error side naming and call it {{successMessage}}